server/cli/launch.py

```diff
@@ -22,7 +23,11 @@ def common_args(func):
     """
     Decorator to contain CLI args that will be common to both CLI and GUI: title and engine args.
     """
+
     @click.option("--title", "-t", help="Title to display (if omitted will use file name).")
+    @click.option("--about",
+                  help="A URL to more information about the dataset."
+                       "(This must be an absolute URL including HTTP(S) protocol")
```

Comment:
Optional: suggest running this help text by @liaprins-czi - it is clear, but perhaps could be worded more gracefully?

Reply:
possible:   "A absolute URL providing more information about this dataset."

Reply:
@liaprins-czi pointed out a missing parenthesis